mpf/modes/carousel/code/carousel.py

```diff
@@ -67,6 +72,15 @@ def mode_start(self, **kwargs):
         self._highlighted_item_index = 0
         self._update_highlighted_item(None)
 
+        # If set to block next/prev on flipper cancel, set those event handlers
+        if self._block_on_flipper_cancel:
+            self._register_handlers(
+                ["flipper_cradle", "flipper_cancel"],
```

Comment:
Those could also become an option

Reply:
Good call. It'd have to be a finite set of options, since the cradle/cancel have different cancellation events, but good plan.